apko-snapshot/action.yaml

```diff
@@ -14,10 +14,17 @@ inputs:
 
   base-tag:
     description: |
-      The base tag to use for publishing the image.
+      The container repository into which we should publish images.
```

Comment:
This sounds like we should rename `base-tag` to `repo` or something? (not necessarily in this PR)

Reply:
Yeah, when I dug in, I immediately hated the input name, but we'll have to sort out staging the signature change.